app/src/config/constants/sub/rule-types.js

```diff
@@ -211,7 +211,7 @@ const RULE_TYPES_CONFIG = {
     HIDE_IN_EXTENSION: false,
     SHOW_DELETE_PAIR_ICON_ON_SOURCE_ROW: false,
     ALLOW_APPLY_RULE_TO_ALL_URLS: true,
-    ALLOW_REQUEST_SOURCE_FILTERS: false,
+    ALLOW_REQUEST_SOURCE_FILTERS: true,
```

Comment:
What is this for? and why did we change it to true now?

Reply:
This key decides if source filters will be visible in the particular rule type or not. It was false for script rule

Reply:
Got it. Name is little misdirecting though